src/components/List/_List.scss

```diff
@@ -10,6 +10,53 @@
 .list {
   margin: unset; // browser automatically applies margin
 
+  &__definition-list {
+    border-bottom: 1px solid var(--ui-gray-light);
+
+    h3 {
+      border-top: 3px solid var(--section-research-primary);
+      padding-top: var(--space-xs);
+    }
+
+    dl {
+      display: grid;
+      gap: 0px;
```

Comment:
I'm super new to grid but wanted to know what's the purpose of setting gap here?  It looks like it's not supported in safari but I don't see a visual difference between how it looks in sfarai and on ff?

Reply:
It is supported--I think you might have been looking at the _flexbox_ gap property.  You are right though that it appears not to be necessary here since 0px is the default setting for the grid gap property.  I went ahead and removed it.